email-utils.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable camelcase */
```

Comment:
Same here. Looks like we have 3 standard ways now of disabling the `camelcase` rule for a line, file, folder. I vote we just disable the rule globally and remove these.